snapshots/forests/2024-05-08/ifl.py

```diff
@@ -0,0 +1,25 @@
+"""Script to create a snapshot of dataset."""
+
+from pathlib import Path
+
+import click
+
+from etl.snapshot import Snapshot
+
+# Version for current snapshot dataset.
+SNAPSHOT_VERSION = Path(__file__).parent.name
+
+
+@click.command()
+@click.option("--upload/--skip-upload", default=True, type=bool, help="Upload dataset to Snapshot")
+@click.option("--path-to-file", prompt=True, type=str, help="Path to local data file.")
```

Comment:
When creating a snapshot from a local file, it's convenient to explain where the file came from. I see that you mentioned it in the metadata file, but maybe it's more useful to do that in the docstring at the top of the script (up to you).